recipes/web3/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "web3" %}
+{% set version = "5.12.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f378a773886122d2126d7ba81e9f5b6d2e7b4def7e88543699bf85a951835942
+
+build:
+  number: 0
+  skip: true  # [win or py==38]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools-markdown
+  run:
+    - python
+    - eth-abi >=2.0.0b6,<3.0.0
+    - eth-account >=0.5.2,<0.6.0
+    - eth-hash >=0.2.0,<1.0.0
+    - eth-typing >=2.0.0,<3.0.0
+    - eth-utils >=1.8.4,<2.0.0
+    - hexbytes >=0.1.0,<1.0.0
+    - ipfshttpclient >=0.4.13,<1
+    - jsonschema >=3.2.0,<4.0.0
+    - lru-dict >=1.1.6,<2.0.0
+    - protobuf >=3.10.0,<4
+    # - pypiwin32 >=223  # [win]
```

Comment:
do you want to try windows builds now?

Reply:
no. they need pypiwin32 which is just a link to pywin32 but not on conda. In order not to patch upstream requirements, I asked in the pywin32 feedstock if they could make an empty link package as a separate output, but no reply yet.

Reply:
Is this just something you saw in their setup.py file? If so, those requirements are effectively ignored during install.

Lets try building and see what happens?

You can depend on `pywin32` and see what happens.

Add something to this effect as a comment, and you can cross reference your upstream issue as well.

Reply:
yes me and grayskull saw it. But also check pypiwin32 myself. (I mean the package, not what the code does as my windows access is limited)